src/popover/src/Popover.js

```diff
@@ -60,11 +62,17 @@ const Popover = memo(
      * Methods borrowed from BlueprintJS
      * https://github.com/palantir/blueprint/blob/release/2.0.0/packages/core/src/components/overlay/overlay.tsx
      */
-    const bringFocusInside = () => {
+    const bringFocusInside = useCallback((e) => {
+      if(isShown && e) {
```

Comment:
Good! Making sure this has the `&& e` patch.

Reply:
👍

Reply:
The formatting for this still feels weird (should be space after `if`), but nbd